arcgis-ios-sdk-samples/Layers/Display dimensions/DisplayDimensionsViewController.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayDimensionsViewController: UIViewController {
+    // MARK: Storyboard views
+    
+    /// The map view managed by the view controller.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Set the viewpoint to the southwest of Edinburgh, UK.
+            mapView.setViewpointCenter(AGSPointMakeWGS84(55.908, -3.305))
```

Comment:
According to the new change in `common-samples/pull/3230`, this line might not be needed. Will verify.